### ISSUE TITLE
Fails to load managed JavaFX dependencies for tests only.

### ISSUE THREAD

**Issue by hmf** [com-lihaoyi/mill#1406]
This is possibly a follow-up for issue #775. When I execute a *standard* class the application runs flawlessly. However, when I attempt to run an MUnit, test I get the following:
```
31/60] managed.compile 
[info] compiling 3 Java sources to /home/hmf/VSCodeProjects/javaFXMill/out/managed/compile/dest/classes ...
[info] done compiling
[48/60] managed.test.resolvedIvyDeps 
1 targets failed
managed.test.resolvedIvyDeps Failed to load source dependencies
  not found: https://repo1.maven.org/maven2/org/openjfx/javafx-controls/16/javafx-controls-16-${javafx.platform}.jar
  not found: https://repo1.maven.org/maven2/org/openjfx/javafx-graphics/16/javafx-graphics-16-${javafx.platform}.jar
  not found: https://repo1.maven.org/maven2/org/openjfx/javafx-base/16/javafx-base-16-${javafx.platform}.jar
```
I have created a test whose source is [here](https://github.com/hmf/javaFXMill/tree/hf_1). If you clone the repository and execute:

``` 
./mill -i managed.runMain helloworld.HelloWorld
```
a JavaFX/OpenFX dialog box with a button will open up as expected. You can close the window to terminate. Now if you execute:

```
./mill -i managed.test managed.PlotSpec.*
```
you get the error reported above.  I suspect this behavior is incorrect. I tested this on Linux, JDK 11, and the latest Mill commit 0.9.8-9-1cfa51. 

The module is defined [here](https://github.com/hmf/javaFXMill/blob/hf_1/build.sc#L43). Note that I even tried to add the libraries again at [this point](https://github.com/hmf/javaFXMill/blob/hf_1/build.sc#L146), but as expected, that did not work. 

If need be, I can do some additional tests to help diagnose the issue. 

TIA.

**Comment by lefou** [comment]
Your test module is probably not using the same `resolutionCustomizer`. We probably should use the same customizer in the test as in the parent module by default.

**Comment by hmf** [comment]
@lefou  You are correct. Added the override to the inner test object and it work. Thank you.

**Comment by lefou** [comment]
See PR #1409
